docs/2fa/developers/faq.md

```diff
@@ -0,0 +1,5 @@
+# Frequently Asked Questions
+
+### **1. How can I integrate with the Automata 2FA protection?**
+
+You can refer to smart contract part to see how to integrate with your contract, please make sure all assets ownership transfer operations should add **only2FA** modifier.
```

Comment:
You can refer to the smart contract section for more technical details. Do ensure that all asset transfer operation include a **only2FA** modifier.